content/docs/components/hyperparameter.md

```diff
@@ -4,25 +4,91 @@ description = "Using Katib to tune your model's hyperparameters"
 weight = 5
 +++
 
-## Using Katib
+Hyperparameter Tuning on Kubernetes.
+This project is inspired by [Google vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf). Katib is a scalable and flexible hyperparameter tuning framework and is tightly integrated with kubernetes. Also it does not depend on a specific Deep Learning framework (e.g. TensorFlow, MXNet, and PyTorch).
+
+## Installing Katib
+
+For running Katib jobs, you have to install necessary packages.
+
+In your Ksonnet app root, run the following
 
-Currently we are using port-forwarding to access the katib services.
-kubernetes version 1.9~
 ```
-kubectl -n kubeflow port-forward svc/katib-ui 8000:80
+export KF_ENV=default
+ks env set ${KF_ENV} --namespace=kubeflow
+ks registry add kubeflow github.com/kubeflow/kubeflow/tree/master/kubeflow
 ```
-~1.8
+
```

Comment:
Let's give the reader a hint about ksonnet, in case they haven't used it before:

```
You can read more about Kubeflow's use of ksonnet in the [ksonnet component guide](/docs/components/ksonnet/).
```